app/utils/push_notifications.js

```diff
@@ -50,7 +50,7 @@ const onRegisterDevice = (data) => {
 };
 
 const loadFromNotification = async (notification) => {
-    await store.dispatch(loadFromPushNotification(notification));
+    await store.dispatch(loadFromPushNotification(notification, app.startAppFromPushNotification));
```

Comment:
Similar to an earlier comment, why would we be calling `loadFromNotification` if we're not starting from a push notification?

Reply:
see above answer